### ISSUE TITLE
navigation doesnt expand

### ISSUE THREAD

**Issue by Krangth** [tmcw/docbox#69]
I'm having an issue where the navigation doesnt expand. For example, in the demo site, if you clicked "wobble" then all the wobble end points would appear. This is only works for me if I am loading the site locally from budo. The same is true for the "show examples in" ui element in the top right where you choose curl, cli, etc. I am serving the website from iis. Is there any advice on this topic?

**Comment by tmcw** [comment]
It's hard to debug without a live example - my best guess is that the JavaScript part of docbox isn't loaded or included, whether because the server is misconfigured, the file wasn't transferred, some browser restriction, or something else.

**Comment by Krangth** [comment]
I'll put a copy in a publicly accessible location when I have enough time. Is bundle.js supposed to be empty?

**Comment by tmcw** [comment]
Nope, it's definitely not supposed to be empty.

**Comment by Krangth** [comment]
I started over and walked through the set up again. Previously, what happened was I got an error using "npm run build"
`'NODE_ENV' is not recognized as an internal or external command,
operable program or batch file.`

After googling, I found several results that suggested putting "SET" at the start of the line which created another problem that didn't produce an error message. This is my first time using npm. On Windows, it is apparently not enough to add SET, you will need to add & somewhere else in the command as well. 

I installed the package I found here and the command seems to have run correctly as the bundle.js file is no longer empty and the navigation works.
https://stackoverflow.com/a/40967643/2565663

However, the top elements are not positioned correctly. https://puu.sh/z9SH2/2770579d50.png
Here is a 7zip of the docbox directory https://drive.google.com/open?id=1LnfUmK_T8xusVFt7KLAZEaIqSn5NVQSe
